reddit_gold/controllers.py

```diff
@@ -79,12 +70,6 @@ class GoldApiController(RedditController):
                    VGold(),
                    VModhash(),
                    deal=VLength('deal', 100))
-    def POST_claim_gold_partner_deal_code(self, responder, deal):
```

Comment:
Need to remove the `@json_validate` above here and all the arguments to it here as well, that's related to this function. If you see something starting with `@` just above a function definition, that's called a "decorator" and changes the function below it, so it's effectively part of the function.